assemblies/resources/bin/setenv

```diff
@@ -35,4 +36,9 @@
 # export KARAF_REDIRECT    # Enable/set the std/err redirection when using bin/start
 
 export EXTRA_JAVA_OPTS="-Dorg.eclipse.jetty.server.Request.maxFormContentSize=1500000 -Dfile.encoding=UTF-8"
+export JAVA_MAX_MEM=1G
+export JAVA_PERM_MEM=256M
```

Comment:
Does this not make the file identical to the other ones? In that case, we could just drop the other files.

Reply:
no it sources bin/check_ports in its last line

Reply:
But the others also contain that line. I think you might have misunderstood me. Adding the `…MEM…` lines, the file `assemblies/resources/bin/setenv` is now identical to `assemblies/dist-develop/src/main/resources/bin/setenv` and `assemblies/dist-allinone/src/main/resources/bin/setenv`. Before these have been special cases which would override the default `setenv`. This does not make any sense any longer if bow all files contain the same content.

Reply:
You are right they were in for some reason before

Reply:
Generally, I wouldn't mind adding those settings to all distributions in which case we should just drop the *special-case*  files. We just shouldn't have several copies of the same identical file overwrite each other ;-=

Reply:
just updated (it was in your version before ;-) )